pkgs/servers/pulseaudio/default.nix

```diff
@@ -9,7 +9,7 @@
 
 , x11Support ? false
 
-, useSystemd ? stdenv.isLinux
+, useSystemd ? (stdenv.isLinux && (!stdenv.hostPlatform == stdenv.buildPlatform || libOnly))
```

Comment:
If I understand this correctly, the check for cross-compiling was only there to avoid mass rebuilds.  Can we please drop it if so, since you're targeting staging anyway?

Reply:
Absolutely, my bad @alyssais